### ISSUE TITLE
Change the CAMERA temperature keywords to add Unfilter (TU_?) to the keyword name

### ISSUE THREAD

**Issue by bberkeyU** [NCAR/ucomp-pipeline#221]
For the Cropico temperatures, we report "T_??????" as the lowpass filtered temperature value and "TU_????" as the raw temperature value. However, for the cameras, we only report the raw temperature and report it as "T_?????".

Should we change the camera temperatures to a "TU_????" format since these are unfiltered temperatures?  If we leave things as they are, we should document the inconsistency in the user guide. If we change to "TU_", we will need to change the header_config.ini for future data, and create an epoch change for all data taken before 2023.

Current Keywords from 2022-09-01 reprocessed November 16,2023.

```
T_RACK  =               28.981 / [C] Computer Rack Temp                         
T_LCVR1 =               34.572 / [C] Lyot LCVR1 Temp                            
T_LCVR2 =               34.497 / [C] Lyot LCVR2 Temp                            
T_LCVR3 =               34.576 / [C] Lyot LCVR3 Temp                            
T_LNB1  =               34.571 / [C] LiNb1 Temp                                 
T_MOD   =               32.737 / [C] Modulator Temp                             
T_LNB2  =               34.499 / [C] LiNb2 Temp                                 
T_LCVR4 =               34.501 / [C] Lyot LCVR4 Temp                            
T_LCVR5 =               34.381 / [C] Lyot LCVR5 Temp                            
T_BASE  =               31.123 / [C] Instrument Baseplate Temp                  
TU_RACK =               30.357 / [C] Computer Rack Temp Unfiltered              
TU_LCVR1=               34.574 / [C] Lyot LCVR1 Temp Unfiltered                 
TU_LCVR2=               34.497 / [C] Lyot LCVR2 Temp Unfiltered                 
TU_LCVR3=               34.576 / [C] Lyot LCVR3 Temp Unfiltered                 
TU_LNB1 =               34.571 / [C] LiNb1 Temp Unfiltered                      
TU_MOD  =               32.746 / [C] Modulator Temp Unfiltered                  
TU_LNB2 =               34.501 / [C] LiNb2 Temp Unfiltered                      
TU_LCVR4=               34.507 / [C] Lyot LCVR4 Temp Unfiltered                 
TU_LCVR5=               34.401 / [C] Lyot LCVR5 Temp Unfiltered                 
TU_BASE =               31.146 / [C] Instrument Baseplate Temp Unfiltered       
T_C0ARR =                4.992 / [C] Camera 0 Sensor array temp                 
T_C0PCB =               34.000 / [C] Camera 0 PCB board temp                    
T_C1ARR =                5.025 / [C] Camera 1 Sensor array temp                 
T_C1PCB =               33.500 / [C] Camera 1 PCB board temp
```

#### Tasks

- [x] change name and comment for the 4 keywords
- [x] add comment line(s) before temperature explaining "Unfiltered"

**Comment by jburkepile** [comment]
Steve has the most knowledge about this and should weigh in.

**Comment by detoma** [comment]
This is a question for Steve.

**Comment by bberkeyU** [comment]
Steve should weigh in if he wants to, and he is assigned this ticket.  But I suspect he may not have a strong opinion on this, so I added more of the team to the ticket.  

My full proposal is to change the header_config.ini file to change the keyword name and comment for future data and to create an epoch to fix the past keywords when we run the L1 pipeline.   Changes would look like the following:
T_C0ARR = 4.992 / [C] Camera 0 Sensor array temp
T_C0PCB = 34.000 / [C] Camera 0 PCB board temp
T_C1ARR = 5.025 / [C] Camera 1 Sensor array temp
T_C1PCB = 33.500 / [C] Camera 1 PCB board temp
to
TU_C0ARR = 4.992 / [C] Camera 0 Sensor array temp Unfiltered
TU_C0PCB = 34.000 / [C] Camera 0 PCB board temp Unfiltered 
TU_C1ARR = 5.025 / [C] Camera 1 Sensor array temp Unfiltered
TU_C1PCB = 33.500 / [C] Camera 1 PCB board temp Unfiltered

It will break Mike and Steve's mission plotting code since they would be looking for the old keywords; I don't think anyone else does anything with these values.  But it will simplify the documentation and remove the inconsistency in reporting UCoMP temperatures.

**Comment by detoma** [comment]
I support what Ben is suggesting. The time to fix these kind of things is now or never.

Is this also a change in the raw data? If so, can we read and modify the header for the existing raw data so everything is consistent? We are not supposed to change raw data but if we do this for the commissioning period is probablhy okay. We are not changing the data value, only the keywords.

**Comment by bberkeyU** [comment]
*IF* we change the raw data we should also make the #220 LCVRELX header comment change while we are in there mucking around.

**Comment by StevenTomczyk** [comment]
I suppose using TU for the camera temps is more consistent, but there is no filtered camera temperature, so I do not think it is confusing to leave it the way it is. If people feel strongly about this, then change it, but I think it is OK to leave it the way it is.  
Definitely change the comment on LCVRRELX.

**Comment by mgalloy** [comment]
I have an epoch file value which is a list of "repair" routines to run on a level 0 file when it is read, which is how I implemented #220. I would add another routine here for the beginning of the mission to fix this issue.

**Comment by bberkeyU** [comment]
COMMENT:  Temapures used in the Lyot filter calibrations are low-pass filtering and reported in keywords that start with T_.  The raw, unfiltered temperature values for recorded temperatures are recorded in keywords that begin with TU_.
